lib/stm32/common/st_usbfs_core.c

```diff
@@ -200,15 +236,33 @@ void st_usbfs_ep_nak_set(usbd_device *dev, uint8_t addr, uint8_t nak)
 uint16_t st_usbfs_ep_write_packet(usbd_device *dev, uint8_t addr,
 				     const void *buf, uint16_t len)
 {
+	void *usb_buf;
+
 	(void)dev;
 	addr &= 0x7F;
 
-	if ((*USB_EP_REG(addr) & USB_EP_TX_STAT) == USB_EP_TX_STAT_VALID) {
-		return 0;
+	if ((GET_REG(USB_EP_REG(addr)) & USB_EP_TYPE) == USB_EP_TYPE_ISO) {
+		/* Isochronous endpoints always use double buffering.
+		 * Additionally, TX_STAT check doesn't make sense for them. */
+		if (GET_REG(USB_EP_REG(addr)) & USB_EP_TX_DTOG) {
+			// buffer 1 is in use by the peripheral
```

Comment:
only C style comments.  Run "make stylecheck"

Reply:
Sure. Remnants of the original PR :)